File: problemas/iniciante/1131/1131.dart
import 'dart:io';

void main() {
  int inter = 0;
  int gremio = 0;
  int empate = 0;
  while (true) {
    List<int> entrada = stdin
        .readLineSync()!
        .trim()
        .split(' ')
        .map((x) => int.parse(x))
        .toList();
    int i = entrada[0];
    int g = entrada[1];

    if (i < g) {
      gremio += 1;
    } else if (i == g) {
      empate += 1;
    } else {
      inter += 1;
    }

    int codigo = 0;
    while (codigo != 1 && codigo != 2) {
      print('Novo grenal (1-sim 2-nao)');
      codigo = int.parse(stdin.readLineSync()!);
    }

    if (codigo == 2) {
      break;
    }
  }

  print('${inter + gremio + empate} grenais');
  print('Inter:${inter}');
  print('Gremio:${gremio}');
  print('Empates:${empate}');

  if (inter > gremio && inter > empate) {
    print('Inter venceu mais');
  } else if (gremio > inter && gremio > empate) {
    print('Gremio venceu mais');
  } else {
    print('Nao houve vencedor');
  }
}

File: problemas/iniciante/1131/1131.jl
using Printf

inter = 0
gremio = 0
empate = 0

while true
  global inter
  global gremio
  global empate
  
  entrada = split(readline())
  
  i = parse(Int32, entrada[1])
  g = parse(Int32, entrada[2])

  if i > g
    inter += 1
  else
    if g > i
      gremio += 1
    else
      empate += 1
    end
  end
  
  codigo = 0
  while codigo != 1 && codigo != 2
    @printf "Novo grenal (1-sim 2-nao)\n"
    codigo = parse(Int32, readline())
  end
  if codigo == 2
    break
  end
end

@printf "%d grenais\n" (inter + gremio + empate)
@printf "Inter:%d\n" inter
@printf "Gremio:%d\n" gremio
@printf "Empates:%d\n" empate
if inter > gremio && inter > empate
  @printf "Inter venceu mais\n"
else
  if gremio > inter && gremio > empate
    @printf "Gremio venceu mais\n"
  else
    @printf "Nao houve vencedor\n"
  end
end